src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/test_appconfig_commands.py

```diff
@@ -102,6 +102,16 @@ def test_azconfig_mgmt(self, resource_group, location):
                          self.check('sku.name', sku),
                          self.check('encryption.keyVaultProperties.keyIdentifier', keyvault_uri.strip('/') + "/keys/{}/".format(encryption_key))])
 
+        self.cmd('appconfig update -n {config_store_name} -g {rg} --disable-local-auth',
```

Comment:
Can you also add a few more test cases like:
- enable local auth after disabling it
- disable local auth during store creation
- Check error message if someone tries to use HMAC auth after disabling local auth
- Check the default value of `disableLocalAuth` if someone doesn't explicitly disable local auth (is it null or false?)

Reply:
Tests added